documentation/book/con-overview-of-kafka-bridge.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+[id='con-overview-of-kafka-bridge-{context}']
+= Overview of the {ProductName} Kafka Bridge
+
+The {ProductName} Kafka Bridge provides a REST API for integrating HTTP-based clients with a Kafka cluster running on {ProductPlatformName}. You can use it to integrate clients that do not have a Kafka client library and so cannot transfer data using the native Kafka protocol.
+
+The Kafka Bridge API consists of two resources -- `consumers` and `topics` -- that together provide several endpoints for interacting with Kafka consumers and producers. You can use the API to: 
+
+* Send messages to a topic or partition.
+
+* Create and delete consumers.
+
+* Subscribe consumers to topics, so that they start receiving messages from those topics.
+
+* Unsubscribe consumers from topics.
+
+* Assign partitions to consumers.
+
+* Retrieve messages from topics.
+
+* Commit a list of consumer offsets.
+
+* Perform seek operations on consumers, so that they start receiving messages from the first or last offset position, or a given offset position.
```

Comment:
Actually the seek operations are executed "on a topic partition" by consumers but not "on consumers".

Reply:
Changed to: "Seek on a partition, so that the consumer starts receiving messages from the first or last offset position, or a given offset position."